src/Symfony/Component/Translation/TranslatorInterface.php

```diff
@@ -45,7 +45,7 @@ public function trans($id, array $parameters = array(), $domain = null, $locale
      *
      * @throws \InvalidArgumentException If the locale contains invalid characters
      */
-    public function transChoice($id, $number, array $parameters = array(), $domain = null, $locale = null);
+    public function transChoice($id, $number = 1, array $parameters = array(), $domain = null, $locale = null);
```

Comment:
Cahnging the interface to make the parameter optional is a BC break (it forces all implementations to make it optional to respect the signature). so this is a total no-go